### ISSUE TITLE
Serverspec test fails on overlay

### ISSUE THREAD

**Issue by kraeml** [DieterReuter/image-builder-rpi64#28]
This snippet fails on make test
```ruby 
context "Docker service file" do
    let(:stdout) { run_mounted("cat /etc/systemd/system/docker.service.d/overlay.conf").stdout }

    it "Daemon uses overlay storage driver" do
      expect(stdout).to contain("--storage-driver overlay")
    end
end
```

I don't know which changes is happend on rpi64. My this could remove?

**Comment by DieterReuter** [comment]
Wait, I've already seen this test case failing. Will provide a fix soon.

**Comment by DieterReuter** [comment]
Will be fixed with #29.

**Comment by kraeml** [comment]
Works fine. Thanks
